Makefile

```diff
@@ -0,0 +1,5 @@
+build:
+	go build -v ./...
+
+test:
+	go test -v ./...
```

Comment:
This project is simple enough that it doesn’t need a make file. If you want to build, you just do `go build`. If you want to test, you can just do `go test`.

I think, maybe, we don’t this file.

Reply:
ya; that's fair